src/storage/test/TestStorage.cpp

```diff
@@ -249,19 +249,19 @@ TEST(StorageTest, DISABLED_TestGcsSync) {
 // to run this test:
 // replace the URL, bucket/container, user name and secret with your company's settings
 // and remove "DISABLED_" from the test name to enable it
-TEST(StorageTest, DISABLED_TestAzureDataLake) {
+TEST(StorageTest, TestAzureDataLake) {
```

Comment:
could you DISABLE it again before check-in?

After the build, we could run `make test` in the build folder to check all the tests, this needs to be disabled not included in the normal runs.